client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-item.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import RefundDialog from './label-refund-modal';
+import ReprintDialog from './label-reprint-modal';
+import TrackingLink from './tracking-link';
+import { openRefundDialog, openReprintDialog } from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+
+class LabelItem extends Component {
+	renderRefund = ( label ) => {
+		const { orderId, siteId, translate } = this.props;
+
+		const today = new Date();
+		const thirtyDaysAgo = new Date().setDate( today.getDate() - 30 );
+		if ( ( label.usedDate && label.usedDate < today.getTime() ) || ( label.createdDate && label.createdDate < thirtyDaysAgo ) ) {
```

Comment:
Can `label.usedDate` be in the future for any reason? I don't think so, `&& label.usedDate < today.getTime()` seems redundant.